webapp/components/UploadUserDataProgressBar.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Center, Circle, Flex, Progress, Wrap } from "@chakra-ui/react";
+import { PropsWithChildren } from "react";
+
+import theme from "~~/styles/theme";
+
+const PointAndLabel = ({
+  label,
+  position,
+  progress,
+}: {
+  label: string;
+  position: number;
+  progress: number;
+}) => {
+  return (
+    <>
+      <Circle
+        bg="white"
+        size={"24px"}
+        position="absolute"
+        left={`${position}%`}
+        marginX={"-12px"}
+        zIndex={1}
+        borderColor={
+          progress < position ? "#A7A7A7" : theme.colors.primary["500"]
```

Comment:
don't need to import the theme, you can do 'primary.500' and chakra will find it